### PR TITLE
Jarv/ora conf

### PR DESCRIPTION
**Pull request by jarv**
- apt packages for ease and ora moved to an ansible var
- configurable virtualenvs for ease and ora (defaults to /opt/edx)
- upstart configurations moved from repo to ansible

### CODE REVIEW

File: playbooks/roles/ora/tasks/ease.yml [hunk 1]
```diff
@@ -45,29 +45,31 @@
...
+- name: install ease system packages
+  apt: pkg={{item}} state=present
+  with_items: ease_debian_pkgs
   tags:
   - ease
   - deploy
 
-# Install the python pre requirements into {{ venv_dir }}
+
+# Install the python pre requirements into {{ ease_venv_dir }}
```
Comment (feanil) [inline playbooks/roles/ora/tasks/ease.yml:57]: Does ease need its own venv?

File: playbooks/roles/ora/vars/main.yml [hunk 2]
```diff
@@ -4,6 +4,9 @@ ora_code_dir: "{{ app_base_dir }}/edx-ora"
 # Default nginx listen port
 # These should be overrided if you want
 # to serve all content on port 80
+ora_venv_dir: "/opt/edx"
```
Comment (feanil) [inline playbooks/roles/ora/vars/main.yml]: if it is going to default to '/opt/edx' we should continue to use the {{venv_dir}} variable.

ora_venv_dir = {{venv_dir}}

File: playbooks/roles/ora/tasks/ease.yml (hunk 1 above)
Comment (jarv) [inline playbooks/roles/ora/tasks/ease.yml:57]: It has requirements files in the eease repo so I have it pointing to the
ora venv for now.

File: playbooks/roles/ora/vars/main.yml (hunk 2 above)
Comment (jarv) [inline playbooks/roles/ora/vars/main.yml]: fixed

**Comment by feanil** [comment]
looks good.

**Comment by feanil** [comment]
looks good.

**Pull request merged**